tests/common/plugins/conditional_mark/__init__.py

```diff
@@ -307,6 +307,7 @@ def load_basic_facts(session):
 
     results['topo_type'] = tbinfo['topo']['type']
     results['topo_name'] = tbinfo['topo']['name']
+    results['testbed'] = testbed_name
```

Comment:
Can you name the key more specifically, like `tb_name` or `testbed_name`?

Reply:
I use the key name 'testbed' is because the value is a combination of the testbed name and the topology name, which is equal to session.config.option.testbed (like r-tigon-20-t1-64), so I think 'testbed' is more accurate than 'testbed_name' here. What do you think?

Reply:
The `testbed` is much better.